services/app/apps/codebattle/assets/js/widgets/slices/initial.js

```diff
@@ -450,4 +452,5 @@ export default {
       mute: JSON.parse(localStorage.getItem('ui_mute_sound')),
     },
   },
+  leaderboardUsers: initialLeaderboardUsers,
```

Comment:
Давай лучше будет тут весь лидерборд
Типы списков лидеров лучше убрать в отдельный config

```suggestion
  leaderboard: {
    loading: 'initial',
    period: periodTypes.WEEKLY,
    users: initialLeaderboardUsers,
    error: null,
  },
```